src/main/java/seedu/duke/project/Project.java

```diff
@@ -18,26 +16,20 @@
     public Project(String projectName) {
         this.projectName = projectName;
         this.isDone = false;
-        this.tasks = new TaskList();
+        this.tasks = new ArrayList<>();
         this.projectDescription = "";
     }
 
-    public String getStatusIcon() {
-        
-        return (isDone ? "done" : "not done"); //return tick or X symbols
-        
-    }
-
-    public TaskList getTaskList() {
-        return tasks;
+    public Task getTask(int taskIndex) {
+        return tasks.get(taskIndex);
```

Comment:
Im not sure about the rest, but I got confused when i first read the code.
Maybe we can change the name to getTaskInsideAProject()?

Reply:
My thinking is it would be clear where this method is called ie. project.getTask().

What do the rest think?